app/src/main/java/dev/tuna/leechunbok/HomeScreen.kt

```diff
@@ -1,30 +1,64 @@
 package dev.tuna.leechunbok
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 import dev.tuna.leechunbok.ui.theme.LeechunbokTheme
 
 @Preview(showBackground = true)
 @Composable
 fun HomeScreen() {
-    Greeting("Android")
+    MemberButtons(members = TunaMember.values()) { member ->
+        when (member) {
+            TunaMember.EAST_JUNG -> {}
+            TunaMember.NOAH -> {}
+            TunaMember.JUYOUNG -> {}
+            TunaMember.ROBIN -> {}
+            TunaMember.KADE -> {}
+        }
+    }
 }
 
-
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
+private fun MemberButtons(
+    modifier: Modifier = Modifier,
+    members: Array<TunaMember>,
+    onClick: ((TunaMember) -> Unit)? = null
```

Comment:
```suggestion
    onClick: ((TunaMember) -> Unit) = {} 
```

?.{} 를 제거해서 가독성을 높일 수 있습니다.